docs/src/pages/docs/the-accounts-struct.md

```diff
@@ -169,8 +169,6 @@ pub struct Initialize<'info> {
 }
 ```
 
-Note the doc comment needs to be a [line or block doc comment](https://doc.rust-lang.org/reference/comments.html#doc-comments) (/// or /\*\*) to be interepreted as doc attribute by Rust. Double slash comments (//) are not interpreted as such.
```

Comment:
What's the motivation for this change?

Reply:
The same information is duplicated.

<img width="609" alt="image" src="https://user-images.githubusercontent.com/97757225/188079330-f8cd2bff-b2d5-4867-acf1-bec4431fc9c7.png">